### PR TITLE
Chrysler: torque tuning + increased steer rate

### PR DESCRIPTION
**Pull request by nuwandavek**
- Looking at the `JEEP` ramp-up, ramp-down, the steer rate can be 2.5xed without exceeding jerk of 2.5m/s^3. 
- In this PR, it is increased from 3 to 6

### CODE REVIEW

**Comment by ajw0** [comment]
This is a big improvement for my 2020 Jeep Grand Cherokee. There is much less crossing over the lines when going around sharp corners. Here's a quick route: d914c7bc4a716312|2023-01-20--17-16-30--7

**Comment by nuwandavek** [comment]
> This is a big improvement for my 2020 Jeep Grand Cherokee. There is much less crossing over the lines when going around sharp corners. Here's a quick route: d914c7bc4a716312|2023-01-20--17-16-30--7

Yeah, the old tuning is incorrect with the increase in steer rate. I'm moving this to torque controller altogether. Will update this today!

**Comment by adeebshihadeh** [comment]
Let's make sure to get a test route for each affected model:
- [ ] PACIFICA_2017_HYBRID = "CHRYSLER PACIFICA HYBRID 2017"
- [ ] PACIFICA_2018_HYBRID = "CHRYSLER PACIFICA HYBRID 2018"
- [ ] PACIFICA_2019_HYBRID = "CHRYSLER PACIFICA HYBRID 2019"
- [ ] PACIFICA_2018 = "CHRYSLER PACIFICA 2018"
- [ ] PACIFICA_2020 = "CHRYSLER PACIFICA 2020"
- [ ] JEEP_CHEROKEE = "JEEP GRAND CHEROKEE V6 2018"   # includes 2017 Trailhawk
- [ ] JEEP_CHEROKEE_2019 = "JEEP GRAND CHEROKEE 2019" # includes 2020 Trailhawk

File: selfdrive/car/torque_data/override.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ FORD FOCUS 4TH GEN: [.nan, 1.5, .nan]
 COMMA BODY: [.nan, 1000, .nan]
 
 # Totally new cars
-RAM 1500 5TH GEN: [2.0, 2.0, 0.0]
-RAM HD 5TH GEN: [1.4, 1.4, 0.0]
+RAM 1500 5TH GEN: [1.7, 2.0, 0.13]
+RAM HD 5TH GEN: [1.4, 1.4, 0.13]
```
Comment (adeebshihadeh) [inline selfdrive/car/torque_data/override.yaml:25]: probably want to split this out into a separate PR

**Comment by nuwandavek** [comment]
@adeebshihadeh 
Some segments with max allowed steer, and no faults
- CHRYSLER PACIFICA HYBRID 2019
  - ede4a914fcc6fb6d|2023-01-24--20-54-33
  - ede4a914fcc6fb6d|2023-02-03--12-05-52
- JEEP GRAND CHEROKEE 2019
  - d914c7bc4a716312|2023-01-20--17-16-30
  - d914c7bc4a716312|2023-01-20--17-40-51

**Comment by adeebshihadeh** [comment]
> @adeebshihadeh Some segments with max allowed steer, and no faults
> 
> * CHRYSLER PACIFICA HYBRID 2019
>   
>   * ede4a914fcc6fb6d|2023-01-24--20-54-33
>   * ede4a914fcc6fb6d|2023-02-03--12-05-52
> * JEEP GRAND CHEROKEE 2019
>   
>   * d914c7bc4a716312|2023-01-20--17-16-30
>   * d914c7bc4a716312|2023-01-20--17-40-51

Plotted one of those routes and looks like we probably want to lower the gains 10% like RAM?

File: selfdrive/car/chrysler/values.py [hunk 2]
```diff
@@ -56,6 +60,11 @@ def __init__(self, CP):
 RAM_HD = {CAR.RAM_HD, }
 RAM_CARS = RAM_DT | RAM_HD
 
+# the increased steer rate hasn't been verified on these cars.
+# remove from this list once it's been tested and confirmed to not fault
+CHRYSLER_OLD_TUNING_BLACKLIST = {CAR.PACIFICA_2017_HYBRID, CAR.PACIFICA_2018, CAR.PACIFICA_2018_HYBRID,
```
Comment (adeebshihadeh) [inline selfdrive/car/chrysler/values.py:65]: @nuwandavek the cars in this list will have the old steering rate + PID tuning. you can just remove models from the list and merge when they're ready
